### DISCUSSION TITLE
General Mr.31 Notes (hoi Jam :>)

### DISCUSSION THREAD

**Discussion by Mr31YT** [JaWarrior12/constitution-rewrite#6] [Ideas]
1. I have a suggestion to offences. Possibly add some things that can be considered for trial from incidents of a high caliber from the Discord server such as Extreme racism and that kind of stuff. Thats just one note I have so far I'm still reading the article
though now that I think of it that might go under 10

**Comment by JaWarrior12** [comment]
Just added. 1.a.xi. Usage of slurs and making racist remarks.
